src/lib/util/selinux.c

```diff
@@ -369,6 +369,18 @@ selinux_copy_files_safely(struct selinux_safe_copy *table,
     /* First, write content into temporary files, so we can safely fail
      * on error without overwriting destination files. */
     for (i = 0; table[i].source != NULL; i++) {
+        if (file_exists(table[i].source) == ENOENT) {
+            if (!table[i].can_unlink) {
+                ERROR("File [%s] does not exist but it is not safe to "
+                      "unlink [%s].", table[i].source, table[i].destination);
```

Comment:
I understand this error, but for administrators the message may look strange.
Why deleting is not safe when the file is actually gone? Also I would replace s/unlink/remove or delete/.

Maybe something like "File [%s] does not exist in selected profile but it is not safe to delete [%s]."

Reply:
I realized that with full path expanded it is a bit clearer. If you think that the error message is better as it is, let me know.

Reply:
How about now? This is really just a corner case if someone deleted the file from backup. It should not happen.